pkg/client/clientcmd/loader.go

```diff
@@ -86,6 +94,50 @@ func mergeConfigWithFile(startingConfig *clientcmdapi.Config, filename string) e
 	return nil
 }
 
+// resolveLocalPaths resolves all relative paths in the config object with respect to the parent directory of the filename
+// this cannot be done directly inside of LoadFromFile because doing so there would make it impossible to load a file without
+// modification of its contents.
+func resolveLocalPaths(filename string, config *clientcmdapi.Config) error {
```

Comment:
Not a big deal, since these aren't exported, but having `resolveLocalPaths` take a filepath and `resolveLocalPath` take a dirpath is a little unexpected. If the assymetry bothers you, change it :)